libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotificationView.kt

```diff
@@ -0,0 +1,237 @@
+package com.mapbox.navigation.trip.notification
+
+import android.app.PendingIntent
+import android.content.Context
+import android.graphics.Bitmap
+import android.text.SpannableString
+import android.view.View
+import android.widget.RemoteViews
+import androidx.core.content.ContextCompat
+import com.mapbox.navigation.utils.internal.SET_BACKGROUND_COLOR
+
+internal class MapboxTripNotificationView(
```

Comment:
Great to see this logic decoupled from the rest of the codebase, could you separate the tests similarly and add coverage for the parts that weren't testable before (if any)?

Reply:
done